src/login/logind-utmp.c

```diff
@@ -42,20 +42,27 @@ _const_ static usec_t when_wall(usec_t n, usec_t elapse) {
         return left % USEC_PER_HOUR;
 }
 
-bool logind_wall_tty_filter(const char *tty, void *userdata) {
-        Manager *m = userdata;
-        const char *p;
-
-        assert(m);
-
-        if (!m->scheduled_shutdown_tty)
-                return true;
+bool logind_wall_tty_filter(const char *tty, bool is_local, void *userdata) {
+        Manager *m = ASSERT_PTR(userdata);
+        assert(m->scheduled_shutdown_action);
 
-        p = path_startswith(tty, "/dev/");
+        const char *p = path_startswith(tty, "/dev/");
         if (!p)
                 return true;
 
-        return !streq(p, m->scheduled_shutdown_tty);
+        /* Do not information about events which do not destroy local sessions to local terminals. We can
```

Comment:
```suggestion
        /* Do not inform about events which do not destroy local sessions to local terminals. We can
```

or maybe a missing verb before `information`?